Editor/Window/ConnectToFleetInput.cs

```diff
@@ -113,9 +113,9 @@ private void OnCancelButtonClicked()
 
         private async void OnSelectFleetDropdown(string fleetSelection)
         {
-            var items = fleetSelection.Split(" | ");
+            var items = fleetSelection.Split(" (");
```

Comment:
I wasn't a fan of the split the vendors used previously and am less of a fan with the new format. If you're willing, I'd prefer this was a regex and that we had some unit testing around these methods; however, I won't block on this